### PR TITLE
Feature/SOF-4479

### PR DESCRIPTION
**Pull request by adewyer**
Add InChI & InChI Key to properties in esse

### CODE REVIEW

File: example/properties_directory/structural/inchi.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "name": "inchi",
+    "value": "1S/Si2/c1-2"
```
Comment (timurbazhirov) [inline example/properties_directory/structural/inchi.json:3]: This seems like an "inchi" for fcc Si. Should we have it for something else more meaningful? Same for the inchi_key below

Comment (adewyer) [reply example/properties_directory/structural/inchi.json:3]: We can do a small molecule like benzene

Comment (adewyer) [reply example/properties_directory/structural/inchi.json:3]: `inchi = 1S/C6H6/c1-2-4-6-5-3-1/h1-6H`
`inchi_key = UHOVQNZJYSORNB-UHFFFAOYSA-N`

**Comment by adewyer** [comment]
PR was merged after changing the inchi and inchi key examples to reflect the inchi string and inchi key for a benzene molecule.

**Pull request merged**